app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -281,7 +283,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.getUserById(userId)
+        val actualResponse = apiService.getUserById(bearerToken, userId)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver